packages/vue-output-target/package-lock.json

```diff
@@ -12,26 +12,91 @@
         "@stencil/core": "^2.9.0 || ^3.0.0"
       }
     },
-    "node_modules/@stencil/core": {
```

Comment:
It looks like pnpm changes (JEDI day project?) made it in here. Can you revert those?

Reply:
Crazy. Local cache. I'll rm -rf my node_modules dir before re-running...